embrace-android-delivery/src/main/kotlin/io/embrace/android/embracesdk/internal/delivery/storage/FileStorageService.kt

```diff
@@ -9,7 +10,22 @@ import java.io.InputStream
 interface FileStorageService {
```

Comment:
We we paramaterize the key type in case other things that are stored can't be addressed with a `StoredTelemetryMetadata` object?